src/database/migrations/V1__addTest.ts

```diff
@@ -0,0 +1,16 @@
+import knex from 'knex';
+
+const TEST_TABLE = 'test';
+
+export async function up(db: knex) {
+  await db.schema.createTable(TEST_TABLE, (table) => {
+    table.uuid('id').primary();
+    table.uuid('user_id').notNullable();
+    table.uuid('test_type_id').notNullable();
+    table.timestamp('creation_time').notNullable().index();
```

Comment:
We need add jsonb column for the results themselves.  And probably a text field for doctors notes (that's not in our previous specs, but Jack seemed pretty keen on it.

Reply:
Can we also add the tester's userId?  So if someone self-tests it will be the same but, but if a trusted user performs it, we would ideally know both who the test is for and who entered the results.